generate-client.ps1

```diff
@@ -0,0 +1,24 @@
+# Generate the client
+nswag run apiserver-nswag.json
+nswag run oauth-nswag.json
+
+# Remove long namespaces from class names in the generated client
+$client_filepaths = @("./src/APIServer/TokenClient.cs", "src/OAuth/OAuthClient.cs")
```

Comment:
Not sure if you have seen it. I may have commented in the wrong place. Very minor, could you make it consistent: `./src/OAuth/OAuthClient.cs`